### PR TITLE
stm32cube: update stm32h7 cube to v1.6.0

### PR DESCRIPTION
**Pull request by FRASTM**
Update Cube version for STM32H7xx series
on https://github.com/STMicroelectronics/STM32Cube
from version v1.5.0
to version v1.6.0

Signed-off-by: Francois Ramu francois.ramu@st.com

### CODE REVIEW

**Comment by FRASTM** [comment]
clean README

File: stm32cube/stm32h7xx/CMakeLists.txt [hunk 1]
```diff
@@ -19,16 +19,19 @@ zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DAC drivers/src/stm32h7xx_hal_
...
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DFSDM drivers/src/stm32h7xx_hal_dfsdm.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DFSDM_EX drivers/src/stm32h7xx_hal_dfsdm_ex.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DMA drivers/src/stm32h7xx_hal_dma.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DMA2D drivers/src/stm32h7xx_hal_dma2d.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DMA_EX drivers/src/stm32h7xx_hal_dma_ex.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DSI drivers/src/stm32h7xx_hal_dsi.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DTS drivers/src/stm32h7xx_hal_dts.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_ETH drivers/src/stm32h7xx_hal_eth.c)
-zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_ETH_EXT drivers/src/stm32h7xx_hal_eth_ex.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_ETH_EX drivers/src/stm32h7xx_hal_eth_ex.c)
```
Comment (erwango) [inline stm32cube/stm32h7xx/CMakeLists.txt]: This symbol does not exist on Zephyr side. It has to be defined in modules/Kconfig.stm32.
(CONFIG_USE_STM32_HAL_ETH_EXT was not defined either)

File: stm32cube/stm32h7xx/CMakeLists.txt [hunk 2]
```diff
@@ -48,19 +51,24 @@ zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_LTDC_EX drivers/src/stm32h7xx_
...
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_MMC_EX drivers/src/stm32h7xx_hal_mmc_ex.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_NAND drivers/src/stm32h7xx_hal_nand.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_NOR drivers/src/stm32h7xx_hal_nor.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_OPAMP drivers/src/stm32h7xx_hal_opamp.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_OPAMP_EX drivers/src/stm32h7xx_hal_opamp_ex.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_OSPI drivers/src/stm32h7xx_hal_ospi.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_OTFDEC drivers/src/stm32h7xx_hal_otfdec.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_PCD drivers/src/stm32h7xx_hal_pcd.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_PCD_EX drivers/src/stm32h7xx_hal_pcd_ex.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_PSSI drivers/src/stm32h7xx_hal_pssi.c)
```
Comment (erwango) [inline stm32cube/stm32h7xx/CMakeLists.txt]: Symbol USE_STM32_HAL_PSSI to be defined in Zephyr side

File: stm32cube/stm32h7xx/CMakeLists.txt (hunk 1 above)
Comment (FRASTM) [reply stm32cube/stm32h7xx/CMakeLists.txt]: this is added in modules/Kconfig.stm32 with https://github.com/zephyrproject-rtos/zephyr/pull/23259

File: stm32cube/stm32h7xx/CMakeLists.txt (hunk 2 above)
Comment (FRASTM) [reply stm32cube/stm32h7xx/CMakeLists.txt]: this is added in modules/Kconfig.stm32 with https://github.com/zephyrproject-rtos/zephyr/pull/23259

**Comment by erwango** [comment]
Closing, following merge of #47

**Pull request closed without merging**